x-pack/plugins/security_solution/public/flyout/right/hooks/use_accordion_state.ts

```diff
@@ -32,7 +32,7 @@ export interface UseAccordionStateValue {
 }
 
 /**
- * Tiny hook for controlled useAccordionState
+ * Tiny hook for controlled AccordionState
```

Comment:
while making this change to this file, I think you could replace

```typescript
state: typeof CLOSED | typeof OPEN;
```

by 

```typescript
state: ToggleReducerState
```

line 26?